File: Salary/Classes/(ClsInformation).cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
//Salery
namespace Salary
{
    public class ClsInformation
    {
        //public String Version
        //{
        //    get {
        //        string YY = "96";
        //        string MM = "06";
        //        string DD = "26";
        //        string HH = "10";
        //        string mmm = "30";
        //        return YY + MM + DD + HH + mmm;
        //    }
        //}
        public string Version()
        {
            string Ver = System.Reflection.Assembly.GetExecutingAssembly().GetName().Version.ToString();
            Ver = Ver.Replace(".", null);
            return Ver;
        }
        
    }
}

File: UserControls/Classes/(ClsInformation).vb
Public Class ClsInformation
    'UserControl
    Public ReadOnly Property PersianToday
        Get
            Dim Uc As New UserControls.UcDatePicker
            Return Uc.GetPersianDate().Trim()
        End Get
    End Property
    Public ReadOnly Property PersianYear
        Get
            Dim Uc As New UserControls.UcDatePicker
            Return Mid(Uc.GetPersianDate().Trim(), 1, 4)
        End Get
    End Property
    'Public ReadOnly Property Version As String
    '    Get
    '        Dim YY As String = "96"
    '        Dim MM As String = "06"
    '        Dim DD As String = "26"
    '        Dim HH As String = "10"
    '        Dim mmm As String = "30"
    '        Return YY & MM & DD & HH & mmm
    '    End Get
    'End Property
    Public Function Version() As String
        Dim Ver As String = System.Reflection.Assembly.GetExecutingAssembly().GetName().Version.ToString
        Ver = Ver.Replace(".", Nothing)
        Return Ver
    End Function
End Class
